native/c/zjbm.c

```diff
@@ -36,11 +39,36 @@ typedef struct
   unsigned char buffer[SYMBOL_ENTRIES * 16];
 } JSYMBOLO;
 
+typedef struct psa PSA;
+typedef struct cvt CVT;
+typedef struct jesct JESCT;
+
+static void get_ssibssnm(SSIB *ssib)
+{
+  PSA *psa_a = (PSA *)0;
+  CVT *cvt_a = psa_a->flccvt;
+  JESCT *jesct_a = cvt_a->cvtjesct;
+
+  if (NULL != jesct_a->jespjesn)
+  {
+    if (0 == strcmp(jesct_a->jespjesn, "JES3"))
+    {
+      zwto_debug("JES3 environment not supported!");
```

Comment:
I know it's debugging only but reminder to delete `zwto_debug` statements until we add a flag that controls these at runtime

Reply:
Resolved in 95d0833f9037557cceee4441560ba490cf32b1a7